ArticleShapeExtractor/modules/Errors.js

```diff
@@ -1,14 +1,71 @@
-const Container = require("./Container.js");
-const errorFactory = Container.resolve("ErrorFactory");
+class ArgumentError extends Error {
```

Comment:
I would expect something like this, so don't need to use `global`

```js
module.exports = {
    ArgumentError: ArgumentError,
}
```
```js
const { ArgumentError } = require('errors.js');
throw new ConfigurationError(`User Query "${this._userQueryName}" does not exist.`);
```

Reply:
Fixed in c2e7210fc60e144b966735b6012d01d837554484